android/sdk/src/main/java/org/jitsi/meet/sdk/AudioModeModule.java

```diff
@@ -121,6 +169,30 @@ public void run() {
     private int mode = -1;
 
     /**
+     * Audio device types.
+     */
+    private static final String DEVICE_BLUETOOTH  = "BLUETOOTH";
+    private static final String DEVICE_EARPIECE   = "EARPIECE";
+    private static final String DEVICE_HEADPHONES = "HEADPHONES";
+    private static final String DEVICE_SPEAKER    = "SPEAKER";
+
+    /**
+     * List of currently available audio devices.
+     */
+    private HashSet<String> availableDevices = new HashSet<>();
```

Comment:
* Do you need `availableDevices` to be of type `HashSet`? If not, please use `Set`.
* If the above is possible, then use `Collections.emptySet()` instead of `new`. The former has a chance of being more optimal than the latter.